HHS/Head-Start-TTADP
# [TTAHUB-1874] Change permission name from "COLLABORATOR_TRAINING_REPORTS" to "POC_TRAINING_REPORTS"

## Description of change

- Change permission name in DB, backend, and UI. 
- Also remove feature flag and instead render URLs and views conditionally

## How to test

Confirm permissions function as they did before change (another change will be incoming to align with the current expectation).

Confirm only those with one of the three appropriate permissions (or admins) see the "Training Reports" menu link

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1874

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
